docs/src/modules/components/ReactRunner.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { useRunner } from 'react-runner';
+
+interface ReactRunnerProps {
+  code: string;
+  scope: {
+    process: {};
+    import: {};
+  };
+  onError: (error: string | null) => {};
+}
+
+// The docs https://github.com/nihgwu/react-runner
+export default function ReactRunner(props: ReactRunnerProps) {
+  const { code, scope: scopeProp, onError } = props;
+
+  let scope = scopeProp;
+
+  if (process.env.NODE_ENV !== 'production') {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    scope = React.useMemo(() => {
+      const handler = {
+        get() {
+          throw new Error(
+            [
+              'A demo tries to access process.x in ReactRunner. This is not supported.',
+              'If you do not need to show the source, you can set "hideToolbar": true to solve the issue.',
```

Comment:
`disableLiveEdit`?

Reply:
I think that hiding the toolbar is more relevant in this context. The developers should probably not be able to open it on codesandbox too if `process` is accessed.